micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnProcessEngineConfiguration.java

```diff
@@ -40,8 +42,11 @@
 
     protected final SynchronousTransactionManager<Connection> transactionManager;
 
-    public MnProcessEngineConfiguration(Configuration configuration, ApplicationContext applicationContext, DataSource dataSource, SynchronousTransactionManager<Connection> transactionManager, ProcessEngineConfigurationCustomizer processEngineConfigurationCustomizer, ArtifactFactory artifactFactory) {
+    private final JobExecutorCustomizer jobExecutorCustomizer;
```

Comment:
private -> protected (analog to transactionManager)

Reply:
Done